Update project management docs (#2219)
alan-turing-institute/the-turing-way: ways_of_working.md

|---|---|---|---|---|
| Kirstie Whitaker | PI | - | Oct 2018 - Present | - |
| Malvika Sharan | Co-lead investigator | - | Jul 2021 - Present | previously: Community manager, Jan 2020 - Jun 2021 (100%) |
| Emma Karoune | Community Manager | 5% | Jan 2021 - Present | - |
| Arron Lacey | Community Manager | 10% | Nov 2021 - Present | - |
| Achintya Rao | Community Manager | 10% | Nov 2021 - Present | - |
| Vicky Hellon | Community Manager | 10% | Nov 2021 - Present | - |
| Carlos Martinez-Ortiz | Netherlands eScience Center - RSE community manager | in-kind contribution | Feb 2020 - Present | - |
| Esther Plomp | Faculty of Applied Sciences at Delft University of Technology - Data Steward | in-kind contribution | Feb 2020 - Present | - |
| Sarah Gibson | 2i2c Open Source Infrastructure Engineer/JupyterHub Community Development | in-kind contribution | - | Prevously: Research Data Scientist, Nov 2018 - Aug 2021 (40%) |
| May Yong | Research Software Engineer | 40% | Oct 2021 - Dec 2021 | - |
| Lydia France | Research Software Engineer | 50% | Oct 2021 - Dec 2021 | - |
| Iain Stanson | Research Software Engineer | 30% | Nov 2021 - Dec 2021 | - |

> | Paul Owoicho | Google Season of Docs - Technical Writer | 100% | Sep 2020 - Dec 2020 | - |

Reviewer: What do you think about formatting this table to be clearer about separating previous folks?

Not a strong opinion, if you think the dates are enough then that's totally fine with me.

Author: For this document, it makes sense to put them in separate tables so people can identify who to contact.

All The Turing Way project members commit to
- making the implicit explicit by documenting their work
- abiding by the Code of Conduct in all community platforms and spaces where they represent _The Turing Way_
- following the contribution guideline and keeping it up to date
- recording any new updates, exceptions or useful knowledge in project management and core documents needed to facilitate collaboration
- dedicating their time and expertise in fixing open issues either directly via GitHub or providing mentorship and support to community members and project contributors
- feedback on issues in open source software used throughout _The Turing Way_ by opening an issue _The Turing Way_ GitHub repo or other open source projects where this issue can be fixed
- documenting and sharing any conversation from closed spaces (such as email, Slack or 1:1 meeting) in a GitHub issue that could be useful for the community or community members in enabling their work in _The Turing Way_ 

## Communication

You can reach out to all members by tagging them on GitHub issues or Pull Requests or on [Slack](https://join.slack.com/t/theturingway/shared_invite/zt-fn608gvb-h_ZSpoA29cCdUwR~TIqpBw).

> You can reach the PI and Co-I of the project through their preferred way of communication:

Reviewer: ```suggestion
You can reach the lead investigators of the project through their preferred way of communication:
```
